Flex relevant docs

JV0812 opened the pull request:
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request is to update/create doc which are impacted due to Flex Pricing, as per the [google sheet comments](https://docs.google.com/spreadsheets/d/1Th6B6VZJWixOpnggLicO4phXehXHd0KmDSldwo-btrI/edit#gid=0).

<!-- Enter the GitHub Issue number or the Jira project and number (e.g., SUMO-12345) -->

## Select the type of change:
<!-- What types of changes does your code introduce? Select the checkbox after creating the PR. -->

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

jpipkin1 commented on docs/api/log-search-estimated-usage.md:
```diff
@@ -10,22 +10,22 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-| JP         | https://api.jp.sumologic.com/docs/#tag/logSearchesEstimatedUsage  |
-| US1        | https://api.sumologic.com/docs/#tag/logSearchesEstimatedUsage     |
-| US2        | https://api.us2.sumologic.com/docs/#tag/logSearchesEstimatedUsage |
+| AU         | https://api.au.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType  |
+| CA         | https://api.ca.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType  |
+| DE         | https://api.de.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType  |
+| EU         | https://api.eu.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType  |
+| FED        | https://api.fed.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType |
+| IN         | https://api.in.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType  |
+| JP         | https://api.jp.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType  |
+| US1        | https://api.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType     |
+| US2        | https://api.us2.sumologic.com/docs/#tag/getLogSearchEstimatedUsageByMeteringType |
```
```suggestion
| AU         | https://api.au.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType  |
| CA         | https://api.ca.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType  |
| DE         | https://api.de.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType  |
| EU         | https://api.eu.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType  |
| FED        | https://api.fed.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType |
| IN         | https://api.in.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType  |
| JP         | https://api.jp.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType  |
| US1        | https://api.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType     |
| US2        | https://api.us2.sumologic.com/docs/#operation/getLogSearchEstimatedUsageByMeteringType |
```

jpipkin1 commented on sidebars.ts:
```diff
@@ -643,30 +643,41 @@ module.exports = {
-        'manage/partitions-data-tiers/searching-data-tiers',
+        {
+          type: 'category',
+          label: 'Flex',
```
```suggestion
          label: 'Flex Pricing',
```

jpipkin1 commented on docs/manage/partitions/flex/flex-pricing-faq.md:
```diff
@@ -0,0 +1,78 @@
+
+## Is the Flex pricing available to existing Cloud Flex customers?
+
+No, it is not available for the existing customer. Flex will become available for existing customers in the near future. Currently, Flex Licencing is the default plan for new customers who sign up with Sumo Logic. 
```
```suggestion
No, it is not available for the existing customers. Flex will become available for existing customers in the near future. Currently, Flex Licencing is the default plan for new customers who sign up with Sumo Logic. 
```

jpipkin1 commented on docs/manage/partitions/flex/flex-pricing-faq.md:
```diff
@@ -0,0 +1,78 @@
+
+## How can I get visibility into our Flex search costs?
+
+Your **Account Overview** page shows the credits your org has consumed for Flex data searches. br/><img src={useBaseUrl('/img/subscriptions/flex-usage-categories.png')} alt="flex-usage-categories" style={{border:'1px solid gray'}} width="800"/>
```
```suggestion
Your **Account Overview** page shows the credits your org has consumed for Flex data searches. <br/><img src={useBaseUrl('/img/subscriptions/flex-usage-categories.png')} alt="flex-usage-categories" style={{border:'1px solid gray'}} width="800"/>
```

jpipkin1 commented on docs/manage/partitions/flex/flex-pricing-faq.md:
```diff
@@ -0,0 +1,78 @@
+
+## How can I optimize Flex data search costs?
+
+There are two ways you can control the cost of running queries data in the Flex:
```
```suggestion
There are two ways you can control the cost of running queries data in Flex:
```

jpipkin1 commented on docs/manage/partitions/flex/flex-pricing-faq.md:
```diff
@@ -0,0 +1,78 @@
+
+* Logically apportion your data, depending on how you plan to query it, into multiple partitions, instead of creating a single large partition. Try to limit queries to a single partition as much as possible. Assume you have three different services sending logs to Sumo Logic. There a couple of approaches you can take:
+    * **Create one partition per service**. All the logs for that service go there. This approach can be leveraged if each team owns a service, and they typically query the logs within their service to find issues.
+    * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if most of the time, the team searches one log type per query. For example, team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
```
```suggestion
    * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if, most of the time, the team searches one log type per query. For example, team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
```

jpipkin1 commented on docs/manage/partitions/flex/flex-pricing-faq.md:
```diff
@@ -0,0 +1,78 @@
+* Logically apportion your data, depending on how you plan to query it, into multiple partitions, instead of creating a single large partition. Try to limit queries to a single partition as much as possible. Assume you have three different services sending logs to Sumo Logic. There a couple of approaches you can take:
+    * **Create one partition per service**. All the logs for that service go there. This approach can be leveraged if each team owns a service, and they typically query the logs within their service to find issues.
+    * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if most of the time, the team searches one log type per query. For example, team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
+* Try to use a restrictive time range, instead of running a query with very long time range. 
```
```suggestion
* Try to use a restrictive time range, instead of running a query with a very long time range. 
```

jpipkin1 commented on docs/manage/partitions/flex/index.md:
```diff
@@ -0,0 +1,53 @@
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Flex pricing delivers the new financial model for log management, with which your businesses can finally centralize, store, and analyze all application, infrastructure, and security data in one place to drive collaboration and unmatched velocity while delivering reliable  and secure digital experiences.
```
```suggestion
Flex pricing delivers a new financial model for log management, in which your businesses can centralize, store, and analyze all application, infrastructure, and security data in one place. This drives collaboration and velocity while delivering a reliable  and secure digital experience.
```

jpipkin1 replied:
That's a long sentence. It's also heavy in marketing language. My markup is only a suggestion.

jpipkin1 commented on docs/manage/partitions/flex/view-partition-details-flex.md:
```diff
@@ -0,0 +1,18 @@
+1. Click the row for a Partition to view its details.<br/><img src={useBaseUrl('img/partitions-data-tiers/view-edit-partition-pane-flex.png')} alt="view-edit-partition-pane.png" style={{border:'1px solid gray'}} width="350"/>
+
+    :::note
+    The information displayed for partitions that contain Cloud SIEM Records varies from other partitions. You can tell if a partition contains Cloud SIEM Records from its name: The names of the Sumo Logic partitions that contain Cloud SIEM Records begin with the string `sec_record_`. Note also that you can’t edit a security partition, or configure data forwarding for it. Cloud SIEM users can sea ...
```
```suggestion
    The information displayed for partitions that contain Cloud SIEM Records varies from other partitions. You can tell if a partition contains Cloud SIEM Records from its name because the names of the Sumo Logic partitions that contain Cloud SIEM Records begin with the string `sec_record_`. Note also that you can’t edit a security partition, or configure data forwarding for it. Cloud SIEM users can search security partitions, as described in [Searching for Cloud SIEM Records in Sumo Logic](/docs/cse/records-signals-entities-insights/search-cse-records-in-sumo).
```

jpipkin1 commented on docs/manage/partitions/index.md:
```diff
@@ -60,55 +60,55 @@ In this section, we'll introduce the following concepts:
 <div className="box-wrapper" >
```
This entire section of cards needs to be re-done to match the contents of the left nav.

Also, all the content above the "## Guides" section should be placed into an "Overview" article.

JV0812 replied:
Currently some sections from this doc is moved to different doc. And we will continue to keep the same doc as a landing page for Data tiers for now, because we have the similar set of page for partitions and flex. Also, some existing folders under Manage has similar approach. May be we can clean these things in future in one shot.

jpipkin1 commented on docs/manage/partitions/data-tiers/index.md:
```diff
@@ -1,10 +1,9 @@
 title: Data Tiers
 description: Data Tiers provide the ability to allocate data to different storage tiers based on frequency of access - Continuous, Frequent, and Infrequent.
 ---
 
```
The content of this article should be placed into a new "Overview" article, and this index article should be updated to contain only a card list of the sub-articles for this section.

JV0812 replied:
Currently some sections from this doc is moved to different doc. And we will continue to keep the same doc as a landing page for Data tiers for now, because we have the similar set of page for partitions and flex.

jpipkin1 reviewed (approved):
Bravo, @JV0812! Very nice work. I'm approving the PR contingent on your addressing my suggestions.

If you are pressed for time, you could work on my suggestions to create new "Overview" articles later.

JV0812 commented:
Thanks a lot for your review @jpipkin1. I have fixed your comments and for the comments related to restructuring of the index page, I have added some comments.
